lib/invisible_captcha/controller_ext.rb

```diff
@@ -74,11 +91,25 @@ def timestamp_spam?(options = {})
 
       return false
     end
-
-    def honeypot_spam?(options = {})
+    
+    def ip_spam?(options ={})
```

Comment:
I think we can simplify a bit this method if we make the changes suggested in https://github.com/markets/invisible_captcha/pull/85/files#r573288133